theurgy.lic

```diff
@@ -476,6 +474,7 @@ class Theurgy
     ######################## SCRIPT ########################
 
     pray
+    ensure_copper_on_hand(600, @hometown)
```

Comment:
I figured that would be the case.  Just wanted to get the pull request in first and figured could move it after.